modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path

```diff
@@ -526,6 +526,11 @@
 	<td>//div[contains(@class,'card d-flex')]//h4[contains(.,'Option')]/..//a</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_CARD_SKU</td>
+	<td>//div[@class="cp-information"]//span[contains(text(),'$key_productSKU')]//following::p[contains(.,'$key_productName')]</td>
+	<td></td>
+</tr>
```

Comment:
Change requested to this locator in #960

Reply:
request done (add curly brackets on locator variables) @lorenzolapescara